src/Atmos/Model/source.jl

```diff
@@ -59,7 +61,15 @@ struct RayleighSponge{FT} <: Source
   zsponge::FT
   "Sponge Strength 0 ⩽ c_sponge ⩽ 1"
   c_sponge::FT
+  "Lateral reference relaxation velocity, u [m/s]"
+  u_ref::FT
```

Comment:
could just make this a static array? 
```suggestion
  u_ref::SVector{3,FT}
```